src/components/Fetch/index.js

```diff
@@ -1,22 +1,10 @@
-import React, {Component} from 'react';
-import Reflux from 'reflux';
-import axios from 'axios';
+import React from 'react';
```

Comment:
You'll need to restore most of the things you deleted here to fix the react/jsx-no-undef errors. Basically each one of those seems to correspond to a message saying '___' is not defined.